### ISSUE TITLE
Making the secrets.py file more comprehensible

### ISSUE THREAD

**Issue by marimeireles** [webcompat/webcompat.com#2645]
In the latest updates you've made some changes to the secrets.py file.

It says in the docs that's necessary to add  your URI:

>1. Add the right values to the repo issues URIs. `ISSUES_REPO_URI = "<user>/<repo>/issues"`. For example, miketaylr's setup needs to say `ISSUES_REPO_URI = "miketaylr/test-repo/issues"`

But in secrets.py file there is no mention of this variable. Would it be a good idea to add a phrase explaining about it and add ```ISSUES_REPO_URI``` variable in secrets.py?

Something like this but about URI:
```SECRET_KEY:
    Secret key for signing cookies. Can be ignored for local testing
    SECRET_KEY = '~*change me*~'
```

Also, it might be helpful to say where you can find your GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET.

I think it could be a source of errors because people might get confused.

**Comment by Auralcat** [comment]
Hi, I set up the development environment this afternoon and was about to ping someone on IRC about the ISSUES_REPO_URI field which was missing in `secrets.py`. Then I added it to the file and everything worked fine until now, except that I haven't tried sending an issue to the test repo through GitHub's API yet.

Since it's a variable field, it could be added to the example file (`secrets.py.example`) with a comment explaining where it's used and when/how to set it.

**Comment by marimeireles** [comment]
@Auralcat glad I could help! :)

**Comment by karlcow** [comment]
about `ISSUES_REPO_URI` I think this is fixed now.
https://github.com/webcompat/webcompat.com/blob/master/docs/dev-env-setup.md#test-repository

about the `GITHUB_CLIENT_*` secrets, does the new doc, fixes it?
https://github.com/webcompat/webcompat.com/blob/master/docs/dev-env-setup.md#store-your-settings

This was fixed by #2652 
@miketaylr made a mistake in the commit number.
